contrib/vagrant/provision-config.sh

```diff
@@ -7,14 +7,14 @@ set -o pipefail
 # Set a useful format for xtrace output
 # export PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
 
-ORIGIN_ROOT=$(
+OS_ROOT=$(
```

Comment:
@marun 

In this PR I had changed this to reflect the `$OS_ROOT` name used in all other Bash scripts -- if @smarterclayton is correct and this change is what caused the networking test to fail, can we change this? Who depends on this name?